app/views/user_case_infos/_form.html.erb

```diff
@@ -29,6 +29,11 @@
   <div class="field">
     <%= f.label :charge_type %><br>
     <%= f.text_field :charge_type, :title => 'Enter Your Charge Type', :class => 'tooltips'  %>
+    <select>
```

Comment:
Would suggest the below to get Rails to recognize the field:
```
  <div class="form-group">
    <%= f.label :charge_type %>
    <%= f.select :charge_type, {}, { include_blank: 'Select Your Charge Type' }, :title => 'Select Your Charge Type', :class => 'form-control tooltips' do  %>
       <option value="(a) It has been two years since I was charged with an eligible misdemeanor but not convicted">(a) It has been two years since I was charged with an eligible misdemeanor but not convicted</option>
       <option value="(b) It has been 5 years I was charged but not convicted with an ineligible misdemeanor...">(b) It has been 5 years I was charged but not convicted with an ineligible misdemeanor...</option>
       <option value="(c) It has been 10 years since I finished parole, probation, prison, or supervised release after being charged with an eligible misdemeanor or eligible felony conviction  or inelegibile">(c) It has been 10 years since I finished parole, probation, prison, or supervised release after being charged with an eligible misdemeanor or eligible felony conviction  or inelegibile</option>
    <% end %>
  </div>
```

That will give you a bootstrap-styled Case conclusion field that has the tooltip helper working.

I also cleaned up some of the language for consistency (used `It has` for everything)